helm/managed/logging/elastic/values.yaml

```diff
@@ -10,6 +10,13 @@ elasticsearch:
   minimumMasterNodes: 1 #default is 2, for minimal systems 1 master node should be adequate
   labels: {"managedby" : "HPCC"}
   clusterHealthCheckParams: "local=true" #local node health status
+  #clusterHealthCheckParams: "wait_for_status=yellow&timeout=50s"
```

Comment:
remove commented out line.

Reply:
Then you need user to change to "green" when replicas is more than 1

Reply:
I don't understand your reply. Are you saying you put line 13 intentionally to document something? 
If that's the case go ahead and add the actual comment.